python_ta/__init__.py

```diff
@@ -36,6 +36,8 @@
 from .reporters import REPORTERS
 from .patches import patch_all
 
+from python_ta.upload import upload_to_server
```

Comment:
This should be a relative import, same as the ones on lines 36-37. In fact, you should put this on line 38 (grouped with those imports).

Reply:
It looks like you missed this comment